src/test/java/com/google/devtools/build/lib/exec/util/SpawnBuilder.java

```diff
@@ -63,27 +62,6 @@ public final class SpawnBuilder {
   private PathMapper pathMapper = PathMapper.NOOP;
   private boolean builtForToolConfiguration;
 
-  /**
-   * A {@link DelegateSpawn} that supports output path mapping as described in {@link
-   * com.google.devtools.build.lib.actions.PathMapper}.
-   *
-   * <p>By overriding {@link #getPathMapper()} - and only in test code - instead of adding an extra
-   * field in {@link SimpleSpawn}, we avoid further pressuring memory on large build graphs.
```

Comment:
> "we avoid further pressuring memory on large build graphs"

I suspect removing this and adding PathMapper into SimpleSpawn might result in a regression for non PathMapper users. Let me run a benchmark, and update this comment with it.

Reply:
Ok. Also benchmark shows no regression. There's an 11 % improvement in the system time, but I think that's a fluke.